concrete-core-fixture/src/fixture/lwe_ciphertext_zero_encryption.rs

```diff
@@ -0,0 +1,143 @@
+use concrete_commons::dispersion::Variance;
+use concrete_commons::parameters::LweDimension;
+use concrete_core::prelude::numeric::Numeric;
+use concrete_core::prelude::{
+    LweCiphertextEntity, LweCiphertextZeroEncryptionEngine, LweSecretKeyEntity,
+};
+
+use crate::fixture::Fixture;
+use crate::generation::prototyping::{
+    PrototypesLweCiphertext, PrototypesLweSecretKey, PrototypesPlaintext,
+};
+use crate::generation::synthesizing::{SynthesizesLweCiphertext, SynthesizesLweSecretKey};
+use crate::generation::{IntegerPrecision, Maker};
+use crate::raw::statistical_test::assert_noise_distribution;
+
+/// A fixture for the types implementing the `LweCiphertextZeroEncryptionEngine` trait.
+pub struct LweCiphertextZeroEncryptionFixture;
+
+#[derive(Debug)]
+pub struct LweCiphertextZeroEncryptionParameters {
+    pub noise: Variance,
+    pub lwe_dimension: LweDimension,
+}
+
+impl<Precision, Engine, SecretKey, Ciphertext> Fixture<Precision, Engine, (SecretKey, Ciphertext)>
+    for LweCiphertextZeroEncryptionFixture
+where
+    Precision: IntegerPrecision,
+    Engine: LweCiphertextZeroEncryptionEngine<SecretKey, Ciphertext>,
+    SecretKey: LweSecretKeyEntity,
+    Ciphertext: LweCiphertextEntity<KeyDistribution = SecretKey::KeyDistribution>,
+    Maker: SynthesizesLweSecretKey<Precision, SecretKey>
+        + SynthesizesLweCiphertext<Precision, Ciphertext>,
+{
+    type Parameters = LweCiphertextZeroEncryptionParameters;
+    type RepetitionPrototypes = (<Maker as PrototypesLweSecretKey<Precision, Ciphertext::KeyDistribution>>::LweSecretKeyProto, );
+    type SamplePrototypes = ();
+    type PreExecutionContext = SecretKey;
```

Comment:
In other fixtures, every associated type is a tuple. Maybe it would be better to stick with it by using `(SecretKey, )` here ?